include/NovelRT.Interop/NrtTypedefs.h

```diff
@@ -71,6 +71,7 @@ extern "C"
 #include "DotNet/NrtDotNetTypedefs.h"
 #endif
 #include "Ecs/Audio/NrtEcsAudioTypedefs.h"
+#include "Ecs/Graphics/NrtEcsGraphicsTypedefs.h"
 #include "Ecs/NrtEcsTypedefs.h"
 #ifdef NOVELRT_INK
 #include "Ink/NrtInkTypedefs.h"
```

Comment:
GH won't let me suggest the change but can we delete the Ink stuff here?

Reply:
Done